core/ui_mixins/quick_panel.py

```diff
@@ -157,6 +157,7 @@ class PaginatedPanel:
     limit = 6000
     selected_index = None
     on_highlight = None
+    is_this_selected = None
```

Comment:
If I understand correctly, we need this because the index number of the selected item would be unknown if a generator is used. Instead of adding one more argument here, I suggest using the existing variable `selected_index`.

We could check if it is a callable. If it is, just do 

```
if selected_index(entry):
    bla lka bla
```